content/applications/finance/accounting/fiscal_localizations/localizations/united_kingdom.rst

```diff
@@ -0,0 +1,84 @@
+==============
+United Kingdom
+==============
+
+Configuration
+=============
+
+:ref:`Install <general/install>` the following modules to get all the features of the UK
+localization:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`UK - Accounting`
+     - `l10n_uk`
+     - Installing this module grants you access to:
+        - CT600-ready chart of accounts
+        - VAT100-ready tax structure
+        - InfoLogic UK counties listing
```

Comment:
```suggestion
        - InfoLogic UK counties listing
```
Couldn't find anything online on "InfoLogic" on Google, however I found something under "ListLogic", which *seems* to list counties in the UK. Might be worth checking with whoever is responsible

Reply:
Info to be confirmed. In the meantime, let's keep it as such.